lib/tokenize.h

```diff
@@ -24,6 +24,7 @@
 #include "config.h"
 #include "errorlogger.h"
 #include "tokenlist.h"
+#include "templatesimplifier.h"
```

Comment:
Please add a forward declaration instead of this `#include`

Reply:
Fixed